Example/src/Example.android.tsx

```diff
@@ -121,55 +132,54 @@ export default class IBeaconExample extends Component {
             if (ind === index) return result;
             // add all other beacons to array
             else {
+              // @ts-ignore
               result.push(val);
               return result;
             }
-          }, [])
+          }, []),
         });
       }
     );
     DeviceEventEmitter.addListener(
       'beaconsDidUpdate',
-      ({ beacons: updatedBeacons, region }) => {
+      ({
+        beacons: updatedBeacons,
+        region,
+      }: {
+        beacons: Array<IBeaconAndroid>;
+        region: RegionType;
+      }) => {
         console.log('beaconsDidUpdate', updatedBeacons, region);
 
         const { beacons } = this.state;
-        updatedBeacons.forEach(updatedBeacon => {
-          const index = beacons.findIndex(beacon =>
+        updatedBeacons.forEach((updatedBeacon) => {
+          const index = beacons.findIndex((beacon) =>
             this._isIdenticalBeacon(updatedBeacon, beacon)
           );
           this.setState({
             beacons: beacons.reduce((result, val, ind) => {
               // replace current beacon values for updatedBeacon, keep current value for others
+              // @ts-ignore
```

Comment:
Same comment as above: can this ts-ignore be avoided by adding generic types to reduce?